packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -208,7 +208,7 @@ enum _PickerColumnType {
 ///  * US-English: `| July | 13 | 2012 |`
 ///  * Vietnamese: `| 13 | Tháng 7 | 2012 |`
 ///
-/// Can be used with [showCupertinoModalPopup] to display the picker modally at
+/// Can be used with [showCupertinoModalPopup] to display the picker modal at
```

Comment:
"modally" was not a typo here. It means "in a modal way". Can you undo this change?

Reply:
Thanks for the review, I have changed it back to `modally` .